eks/cluster-loader/configs/cluster-autoscaler/config.yaml

```diff
@@ -0,0 +1,132 @@
+# Constants
+# Currently set to 1 namespace.
+{{$NAMESPACES := DefaultParam .namespaces 1}}
+{{$PODS_PER_NODE := DefaultParam .podsPerNode 10}}
+{{$QPS := DefaultParam .qps 10}}
+{{$POD_RESOURCE_REQUEST_FUZZ_FACTOR := DefaultParam .fuzzFactor .9}}
```

Comment:
I was just thinking, instead of multiplying by FUZZ_FACTOR, why don't we calculate the SYSTEM_POD_OVERHEAD (kube-proxy + aws_node) and substract it.

Reply:
Oh I'm confused. You're doing both. We either need FUZZ_FACTOR or we can use FREE_CPU/MEM. I'd go with the FREE approach. We can just drop the FUZZ_FACTOR stuff

Reply:
Ah ok that's what you meant. got it